docs/manifest.json

```diff
@@ -267,7 +267,7 @@
         "Multi-session applications",
         "/custom-flows/multi-session-applications"
       ],
-      ["Error Handling", "/custom-flows/error-handling"]
+      ["Error handling", "/custom-flows/error-handling"]
```

Comment:
(if we go with title case as pointed out in the comment above, this probably should be adjusted here)